packages/create-catalyst/src/commands/create.ts

```diff
@@ -55,11 +58,47 @@ function getPlatformCheckCommand(command: string): string {
 
 const telemetry = new Telemetry();
 
-async function handleChannelCreation(cliApi: CliApi) {
+async function handleChannelCreation(bc: Https, cliApi: CliApi) {
   const newChannelName = await input({
     message: 'What would you like to name your new channel?',
   });
 
+  const availableLocales = await getAvailableLocales(bc);
+
+  const storefrontLanguage = await select({
+    message: 'Which default language would you like to set for your channel?',
+    default: 'en',
+    choices: availableLocales,
+    theme: {
+      style: {
+        help: () => colors.dim('(Select locale from the list or start typing the name)'),
+      },
+    },
+  });
+
+  const shouldAddAdditionalLocales = await select({
+    message: 'Would you like to add additional languages?',
+    choices: [
+      { name: 'Yes', value: true },
+      { name: 'No', value: false },
+    ],
+  });
+
+  let additionalLocales: string[] = [];
+
+  if (shouldAddAdditionalLocales) {
+    additionalLocales = await multiSelect({
+      choices: availableLocales.filter(({ value }) => value !== storefrontLanguage),
+      maxSelections: 4,
```

Comment:
Could we follow a similar API to some of the other `@inquirer` prompts? They usually have a `validate` function which might be more extendable if we need to use this prompt in the future.

Reply:
Make sense for me. I will check it

Reply:
The validation triggers once the enter is pressed. I guess it's too late and better to check it once space is pressed. 
But If I change when validation is triggered, it will have the different behaviour in compare with other `@inquirer` prompts.

Reply:
I think that logic is what the other `@inquirer/prompts` do: https://github.com/SBoudrias/Inquirer.js/blob/main/packages/checkbox/src/index.ts#L96C3-L98

Reply:
and it calls here
https://github.com/SBoudrias/Inquirer.js/blob/main/packages/checkbox/src/index.ts#L189

but I would like to check amount of selected choices when user press space button:
```
if (isSpaceKey(key)) {
  ...
  if (maxSelections && items.filter(isChecked).length >= maxSelections) {
    setError(`You can only select up to ${maxSelections} choices`);

    return;
  }

  setItems(items.map((choice, i) => (i === active ? toggle(choice) : choice)));
}
```

Reply:
If we always show help text on the max selections + validating it on enter, I think it's _good enough_. If it's actually crucial to do validation on select too, then we can throw another `await validate()` under the space key as well.